app/cdap/components/hydrator/components/TopPanel/NameAndDescription/styles.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Button, TextField, Tooltip, withStyles } from '@material-ui/core';
+import IconSVG from 'components/shared/IconSVG';
+import styled from 'styled-components';
+
+export const EllipsisText = styled.div`
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+`;
+
+export const PipelineName = styled(EllipsisText)`
+  font-size: 14px;
+  margin-top: 5px;
+  line-height: 17px;
+
+  ${({ errorName }) => errorName && `color: #d15668;`}
+`;
+
+export const PipelineDescription = styled(EllipsisText)`
+  color: gray;
+  font-size: 12px;
+`;
+
+export const CustomTooltip = withStyles(() => {
+  return {
+    tooltip: {
+      fontSize: '13px',
+    },
+  };
+})(Tooltip);
+
+export const ErrorTooltip = withStyles(() => {
+  return {
+    arrow: {
+      color: '#d15668',
+    },
+    tooltip: {
+      background: '#d15668',
+    },
+  };
+})(CustomTooltip);
+
+export const SaveButton = styled(Button)`
+  background-color: #5a84e4;
```

Comment:
Is there a reason for customizing the background color of the button?

Reply:
This is a save button, the original MUI button color is darker than the one we have:

mui button color:
![image](https://user-images.githubusercontent.com/20428112/168663831-061870fe-70ec-4e7d-a6a7-ffd18561a51b.png)

the original save button:
![image](https://user-images.githubusercontent.com/20428112/168664010-7cdf2a3c-1402-44a0-9f3e-15618c2a18d4.png)